vnext/Desktop.IntegrationTests/WebSocketIntegrationTest.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// clang-format off
 #include <CppUnitTest.h>
```

Comment:
why?

Reply:
Current clang-format affects readability of MS Unit test framework macros.

Reply:
I think Vlad might have fixed that. You can exempt certain macros in the clang-format file.

Reply:
This is not yet addressed